IJ 16.3.4 build fails

```
:google-cloud-tools-plugin:ultimate:compileTestJava/mnt/disks/jenkins-workspace/jobs/google-cloud-intellij CI Build (IDEA 2016)/workspace/google-cloud-tools-plugin/ultimate/testSrc/com/google/cloud/tools/intellij/javaee/supportProvider/JavaeeFrameworkSupportProviderTestCase.java:49: error: cannot find symbol
    FrameworkVirtualFileSystem.getJ2EEInstance().cleanup();
                                                ^

```

--- a/google-cloud-tools-plugin/ultimate/testSrc/com/google/cloud/tools/intellij/javaee/supportProvider/JavaeeFrameworkSupportProviderTestCase.java
+++ b/google-cloud-tools-plugin/ultimate/testSrc/com/google/cloud/tools/intellij/javaee/supportProvider/JavaeeFrameworkSupportProviderTestCase.java
@@ -31,16 +31,12 @@ public abstract class JavaeeFrameworkSupportProviderTestCase extends
   public static void deleteApplicationServers() {
     final ApplicationServersManager manager = ApplicationServersManager.getInstance();
     final List<ApplicationServer> servers = manager.getApplicationServers();
-    final ApplicationServersManager.ApplicationServersManagerModifiableModel model = manager.createModifiableModel();
+    final ApplicationServersManager.ApplicationServersManagerModifiableModel model =
+        manager.createModifiableModel();
     for (ApplicationServer server : servers) {
       model.deleteApplicationServer(server);
     }
-    ApplicationManager.getApplication().runWriteAction(new Runnable() {
-      @Override
-      public void run() {
-        model.commit();
-      }
-    });
+    ApplicationManager.getApplication().runWriteAction(model::commit);
   }
 
 

--- a/google-cloud-tools-plugin/ultimate/testSrc/com/google/cloud/tools/intellij/javaee/supportProvider/JavaeeFrameworkSupportProviderTestCase.java
+++ b/google-cloud-tools-plugin/ultimate/testSrc/com/google/cloud/tools/intellij/javaee/supportProvider/JavaeeFrameworkSupportProviderTestCase.java
@@ -20,14 +20,14 @@ import com.google.cloud.tools.intellij.debugger.CloudDebugProcessWatcher;
 
 import com.intellij.ide.util.frameworkSupport.FrameworkSupportProviderTestCase;
 import com.intellij.javaee.appServerIntegrations.ApplicationServer;
-import com.intellij.javaee.module.components.FrameworkVirtualFileSystem;
 import com.intellij.javaee.serverInstances.ApplicationServersManager;
 import com.intellij.openapi.application.ApplicationManager;
 
 import java.util.List;
 
 //todo: this class is copied from javaee_tests module. We need to create a separate artifact from that module and use it instead.
-public abstract class JavaeeFrameworkSupportProviderTestCase extends FrameworkSupportProviderTestCase {
+public abstract class JavaeeFrameworkSupportProviderTestCase extends
+    FrameworkSupportProviderTestCase {
   public static void deleteApplicationServers() {
     final ApplicationServersManager manager = ApplicationServersManager.getInstance();
     final List<ApplicationServer> servers = manager.getApplicationServers();
@@ -46,7 +46,6 @@ public abstract class JavaeeFrameworkSupportProviderTestCase extends FrameworkSu
 
   @Override
   protected void tearDown() throws Exception {
-    FrameworkVirtualFileSystem.getJ2EEInstance().cleanup();
     deleteApplicationServers();
     CloudDebugProcessWatcher.getInstance().removeWatcher();
     super.tearDown();

